recipe/meta.yaml

```diff
@@ -1,27 +1,28 @@
 {% set name = "dask" %}
-{% set version = "2022.2.1" %}
+{% set version = "2022.5.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b699da18d147da84c6c0be26d724dc1ec384960bf1f23c8db4f90740c9ac0a89
+  sha256: 0afd69dd0cd9f838fc0710eda1f3e3333d6603b37e93ded7fac4a51d77566a0f
 build:
   number: 0
-  noarch: python
+  skip: true  # [py<38]
 
 requirements:
   host:
     - python
   run:
     - python >=3.8
-    - bokeh >=2.1.1
+    - bokeh >=2.4.2
     - cytoolz >=0.8.2
     - dask-core {{ version }}
     - distributed {{ version }}
     - jinja2
+    - lz4
```

Comment:
Did I read the [link](https://github.com/dask/distributed/pull/6156) correctly in that it indicates dask is behaving as a meta-package for users and that's why they want to include `lz4`? I'm not comfortable with this. Adding simple things often have side effects (e.g. cmake and ninja where circular dependencies are unnecessarily created).

Reply:
I hesitated too.
What I got is: 
- dask can run without lz4 but lz4 brings better performance. Users may not be aware of this.
- lz4 only has python as run dependency 

https://github.com/AnacondaRecipes/lz4-feedstock
https://github.com/AnacondaRecipes/lz4-c-feedstock

Reply:
I saw @chenghlee comment/clarification, so this is good.

Reply:
`dask` is essentially a meta-package, so it's okay to add lz4 as a dependency; the actual functionality is in `dask-core` and `distributed`. This package is meant to install all the other user-niceties that are captured upstream as pip extras.